mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -1,11 +1,27 @@
 AUI.add(
 	'todo',
 	function(A) {
+		var counterVariable = new A.CharCounter(
```

Comment:
This var is not used anywhere. 

Also:
- constants should be caps
- object properties should not be separated by an empty line
- object brackets should have the same indentation